src/compas_rhino/geometry/mesh.py

```diff
@@ -3,54 +3,37 @@
 from __future__ import division
 
 import Rhino
-
 import compas_rhino
 from compas.datastructures import Mesh
-
-from compas_rhino.geometry._geometry import BaseRhinoGeometry
+from ._geometry import BaseRhinoGeometry
 
 
 __all__ = ['RhinoMesh']
 
 
 class RhinoMesh(BaseRhinoGeometry):
     """Wrapper for Rhino mesh objects.
-
-    Attributes
-    ----------
-    vertices (read-only) : list of point
-        The coordinates of the vertices of the mesh.
-    faces (read-only) : list of list of int
-        Faces defined as lists of references into the list of vertices.
-    vertex_color : list
-        The colors of the vertices.
-        Setting this to ``None`` unsets the vertex colors.
-    border (read-only) : list
-        The GUIDs of the border curves.
     """
 
-    def __init__(self):
-        super(RhinoMesh, self).__init__()
-
-    @property
```

Comment:
it is breaking, yes, but these are very obscure and undocumented parts of the framework, and i doubt it will affect anyone's code other than my own :)

what do you suggest?

Reply:
ok, then let's go ahead